song_rater/views.py

```diff
@@ -13,5 +15,30 @@ class SongCreate(CreateView):
     success_url = '/song-added'
 
 
+class SongList(ListView):
+    model = Song
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        song_table = []
```

Comment:
I would call this `song_list` or just `songs` but it's a bit subjective.  This kinda implies, to me, that were dealing with some kind of  `Table` object, while this is just an ordinary list.

Reply:
Agreed, I changed it.